### ISSUE TITLE
Handle case when seq extraction passes over the 0 coordinate of the genome

### ISSUE THREAD

**Issue by erinhwilson** [erinhwilson/promoter-id-from-rnaseq#2]
https://github.com/erinhwilson/promoter-id-from-rnaseq/blob/b5493da3b14c1dda61141d7470c3c0bf83dbeab9/extract_upstream_regions.py#L131

This should be handled differently - probably split into two parts with the pos part of the window slice and a negative part of the window slice

**Comment by dacb** [comment]
Good catch. Thankfully, at least you know your max error rate from this
issue is tightly bounded by biology.
